.github/workflows/publish.yaml

```diff
@@ -46,6 +46,18 @@ jobs:
           tmp=$(mktemp)
           jq '.version = "${{ steps.clean-tag.outputs.TAG }}"' ./package.json > "$tmp" && mv "$tmp" ./package.json
 
+      - name: Install dependencies
+        run: |
+          npm install
```

Comment:
`npm ci` would be better here to ensure it uses lock file exactly

Reply:
thanks, updated